### DISCUSSION TITLE
Welcome to ThedoorCode Discussions!

### DISCUSSION THREAD

**Discussion by tshimsthedoor** [tshimsthedoor/ThedoorCode#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tshimsthedoor** [comment]
I can not run the code generator on controller 
![image](https://user-images.githubusercontent.com/16034869/168487164-27af9999-0b2e-4dff-8933-bcb3610aa121.png)

**Reply by GordonBeeming** [reply]
Is your solution building at the moment?

**Comment by GordonBeeming** [comment]
For some reason it just works now 😅😎

**Reply by tshimsthedoor** [reply]
Yes it was

**Comment by tshimsthedoor** [comment]
![image](https://user-images.githubusercontent.com/16034869/168894088-0b6218a2-146b-439e-8591-1aa5929680c0.png)
I got some error when I added the identity authentication. I could not run the project anymore.

**Reply by tshimsthedoor** [reply]
This issue was fixed 
![image](https://user-images.githubusercontent.com/16034869/169276163-378067f2-7ddb-41f7-b54a-99499bf0a3a8.png)
